src/phongo_client.c

```diff
@@ -938,10 +819,6 @@ static zval* php_phongo_manager_prepare_manager_for_hash(zval* driverOptions, bo
 	ADD_ASSOC_ZVAL_EX(driverOptionsClone, "autoEncryption", autoEncryptionOptsClone);
 
 	return driverOptionsClone;
-
-ref:
-	Z_ADDREF_P(driverOptions);
```

Comment:
Noted this was moved to `php_phongo_manager_make_client_hash`, where it makes more sense.